include/common/s1ap_structs.h

```diff
@@ -70,7 +70,37 @@ struct esm_sec_info {
 	struct proto_conf proto_config;
 };
 
+/*ENB association information*/
+struct enb_assoc_info {
+	int enb_fd;
+	int inStreamId;
+	int outStreamId;
+	int assoc_max_out_streams;
+	int assoc_max_in_streams;
+	int assocId;
+};
+
+/*NAS MSG IE CODES */
```

Comment:
Is it possible to put the spec reference here ?

Reply:
This enb_assoc_info struct is a local structure to store the sctp association information per enodeB fd. 
No spec reference is present for this.

Reply:
Sorry for confusion. i was asking for NasIeType

Reply:
ok. Added.